pkgs/applications/graphics/awesomebump/default.nix

```diff
@@ -22,7 +22,7 @@ let
       install -D bin-linux/QtnPEG $out/bin/QtnPEG
     '';
   };
-in stdenv.mkDerivation {
+in mkDerivation {
```

Comment:
There's another `mkDerivation` above, not sure whether that also needs replacing.